### ISSUE TITLE
Is WORK_CLOTHING_MAX_PER_SERVICE still a thing?

### ISSUE THREAD

**Issue by lukasbischof** [my-zivi/my-zivi#165]
Is there actually a max amount of work clothing
exepsens given by law?

**Comment by TheFehr** [comment]
Yes it is a law thing: [see here](https://www.zivi.admin.ch/zivi/de/home/zivi-sein/waehrend-eines-einsatzes/entschaedigungen-und-spesen.html)
But there is also a 60.- per Month max
